<launch>
  <arg name="port" default="2" />
  <arg name="rate" default="100" />
  <arg name="is_depth_working" default="true" />

  <node pkg="joy" type="joy_node" name="joystick_driver">
    <param name="dev" value="/dev/input/js$(arg port)" />
  </node>-

  <node pkg="riptide_teleop" type="ps3_controller" name="ps3_controller" output="screen" >
    <rosparam command="load" file="$(find riptide_teleop)/cfg/ps3_command_rates.yaml" />
    <rosparam command="load" file="$(find riptide_controllers)/cfg/attitude_controller_properties.yaml" />
    <rosparam command="load" file="$(find riptide_controllers)/cfg/depth_controller_properties.yaml" />
    <rosparam command="load" file="$(find riptide_controllers)/cfg/max_accel.yaml" />
    <rosparam command="load" file="$(find riptide_controllers)/cfg/vehicle_properties.yaml" />
    <param name="rate" type="int" value="$(arg rate)" />
    <param name="is_depth_working" type="bool" value="$(arg is_depth_working)" />
  </node>
</launch>
